<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-I6MRD32" failures="1" tests="1" name="Test" time="15.4" errors="0" timestamp="2024-07-16T11:32:50 IST">
  <testcase classname="ssquareit.seleniumframework.VerifyInvalidLoginCred" name="invalidLoginAttempt" time="15.142">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [https://tutorialsninja.com/demo/index.php?route=account/account] but found [https://tutorialsninja.com/demo/index.php?route=account/login]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://tutorialsninja.com/demo/index.php?route=account/account] but found [https://tutorialsninja.com/demo/index.php?route=account/login]
at ssquareit.seleniumframework.VerifyInvalidLoginCred.invalidLoginAttempt(VerifyInvalidLoginCred.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginAttempt -->
</testsuite> <!-- Test -->
